phoenicis-javafx/src/main/java/org/phoenicis/javafx/views/mainwindow/apps/ViewApps.java

```diff
@@ -96,6 +100,30 @@ public void showAvailableApps() {
      */
     public void populate(List<CategoryDTO> categories) {
         Platform.runLater(() -> {
+            // set default category icons
+            try{
+                String css = "";
+                for (CategoryDTO category : categories) {
+                    css += "." + category.getName().toLowerCase() + "Button{\n";
+                    css += "-fx-background-image: url('" + category.getIcon() + "');\n";
+                    css += "}\n";
+                }
+                File temp = new File("tempfile.css");
+                temp.delete();
+                temp.createNewFile();
+                BufferedWriter bw = new BufferedWriter(new FileWriter(temp));
+                bw.write(css);
+                bw.close();
+                // apply current theme again to fix hierarchy
+                final String shortName = themeManager.getCurrentTheme().getShortName();
+                final String url = String.format("/org/phoenicis/javafx/themes/%s/main.css", shortName);
+                final URL style = this.getClass().getResource(url);
+                getTabPane().getScene().getStylesheets().clear();
+                getTabPane().getScene().getStylesheets().addAll(temp.toURI().toString(), style.toExternalForm());
```

Comment:
Watch out at which position you add this stylesheet. To make it work as a fallback solution I guess it would be better to insert it in an earlier position.